### PR TITLE
Add Korean Description

### PR DESCRIPTION
**Pull request by JAXPLE**
### Description
* I translated everything.
* If there is a problem, I will attach a discord.
* Discord - `JAXPLE#7623`
(Optional) screenshots / videos

I just uploaded the translated version because there is no image reference in other cases.

### CODE REVIEW

File: src/main/resources/assets/wurst/lang/ko_KR.json [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+  "toast.wurst.nochatreports.unsafe_server.title": "Chat messages can be reported",
+  "toast.wurst.nochatreports.unsafe_server.message": "Messages sent on this server may be reported to Big Brother Mojang and might get your account banned globally",
+  "gui.wurst.nochatreports.ncr_mod_server.title": "\"No Chat Reports\" Server",
+  "gui.wurst.nochatreports.ncr_mod_server.message": "This server uses the \"No Chat Reports\" mod.\n\nYou can join this server if you block chat signatures and disable VanillaSpoof.",
+  "button.wurst.nochatreports.signatures_status": "Signatures: ",
+  "description.wurst.nochatreports.message_is_reportable": "This message has a valid signature and is thus vulnerable to fraudulent chat reports.",
+  "gui.wurst.generic.allcaps_blocked": "BLOCKED",
+  "gui.wurst.generic.allcaps_allowed": "ALLOWED"
```
Comment (Alexander01998) [inline src/main/resources/assets/wurst/lang/ko_KR.json:370]: Hi @JAXPLE,

Thanks for your pull request. I appreciate your effort in translating everything and making Wurst more accessible to Korean-speaking users.

However, it seems that there are a lot of duplicated and untranslated strings at the bottom of the `ko_KR.json` file. Could you please update the file to remove any duplicates and untranslated lines?

Feel free to let me know if you encounter any difficulties or have any questions.

**Review by Alexander01998 — CHANGES_REQUESTED** [review]

**Comment by JAXPLE** [comment]
@Alexander01998 
The requested part has been modified.

**Comment by JAXPLE** [comment]
@Alexander01998 I'm sorry I didn't finish it perfectly.
Thanks for the help.

**Review by Alexander01998 — CHANGES_REQUESTED** [review]
Hey @JAXPLE,

Thanks a lot for your patience and your work on these translations! 👍

I was able to take care of a few issues, but I've also left some comments on other lines that I would like you to take a look at. If you could sort out these remaining issues, we should be all set.

Cheers
-Alexander01998

**Comment by JAXPLE** [comment]
@Alexander01998 

description.wurst.hack.aimassist -> true
description.wurst.hack.airplace -> true
description.wurst.hack.anchoraura -> true
description.wurst.hack.antiafk -> true
description.wurst.hack.antiblind -> true
description.wurst.hack.anticactus -> true
description.wurst.hack.antientitypush -> true
description.wurst.hack.antihunger -> true
description.wurst.hack.antiknockback -> true
description.wurst.hack.antispam -> true
description.wurst.hack.antiwaterpush -> true
description.wurst.hack.antiwobble -> true
description.wurst.hack.arrowdmg -> true
description.wurst.setting.arrowdmg.packets -> true
description.wurst.setting.arrowdmg.trident_yeet_mode -> true
description.wurst.hack.autoarmor -> true
description.wurst.hack.autobuild -> true
description.wurst.hack.autocomplete -> true
description.wurst.hack.autodrop -> true
description.wurst.hack.autoleave -> true
description.wurst.hack.autolibrarian -> true
description.wurst.hack.autoeat -> true
description.wurst.setting.autoeat.target_hunger -> true
description.wurst.setting.autoeat.min_hunger -> true
description.wurst.setting.autoeat.injured_hunger -> true
description.wurst.setting.autoeat.injury_threshold -> true
description.wurst.setting.autoeat.take_items_from -> true
description.wurst.setting.autoeat.eat_while_walking -> true
description.wurst.setting.autoeat.allow_hunger -> true
description.wurst.setting.autoeat.allow_poison -> true
description.wurst.setting.autoeat.allow_chorus -> true
description.wurst.hack.autofarm -> true
description.wurst.hack.autofish -> true
description.wurst.hack.automine -> true
description.wurst.hack.autopotion -> true
description.wurst.hack.autoreconnect -> true
description.wurst.hack.autorespawn -> true
description.wurst.hack.autosign -> true
description.wurst.hack.autosoup -> true
description.wurst.hack.autosprint -> true
description.wurst.hack.autosteal -> true
description.wurst.hack.autoswim -> true
description.wurst.hack.autoswitch -> true
description.wurst.hack.autosword -> true
description.wurst.hack.autotool -> true
description.wurst.hack.autototem -> true
description.wurst.hack.autowalk -> true
description.wurst.hack.basefinder -> true
description.wurst.hack.blink -> true
description.wurst.hack.boatfly -> true
description.wurst.hack.bonemealaura -> true
description.wurst.hack.bowaimbot -> true
description.wurst.hack.buildrandom -> true
description.wurst.hack.bunnyhop -> true
description.wurst.hack.cameradistance -> true
description.wurst.hack.cameranoclip -> true
description.wurst.hack.cavefinder -> true
description.wurst.hack.chattranslator -> true
description.wurst.hack.chestesp -> true
description.wurst.hack.clickaura -> true
description.wurst.hack.clickgui -> true
description.wurst.hack.crashchest -> true
description.wurst.hack.creativeflight -> true
description.wurst.hack.criticals -> true
description.wurst.hack.crystalaura -> true
description.wurst.hack.derp -> true
description.wurst.hack.dolphin -> true
description.wurst.hack.excavator -> true
description.wurst.hack.extraelytra -> true
description.wurst.hack.fancychat -> true
description.wurst.hack.fastbreak -> true
description.wurst.hack.fastladder -> true
description.wurst.hack.fastplace -> true
description.wurst.hack.feedaura -> true
description.wurst.hack.fightbot -> true
description.wurst.hack.fish -> true
description.wurst.hack.flight -> true
description.wurst.hack.follow -> true
description.wurst.hack.forceop -> true
description.wurst.hack.freecam -> true
description.wurst.hack.fullbright -> true
description.wurst.hack.glide -> true
description.wurst.hack.handnoclip -> true
description.wurst.hack.headroll -> true
description.wurst.hack.healthtags -> true
description.wurst.hack.highjump -> true
description.wurst.hack.infinichat -> true
description.wurst.hack.instantbunker -> true
description.wurst.hack.invwalk -> true
description.wurst.setting.invwalk.allow_clickgui -> true
description.wurst.setting.invwalk.allow_other -> true
description.wurst.hack.itemesp -> true
description.wurst.hack.itemgenerator -> true
description.wurst.hack.jesus -> true
description.wurst.hack.jetpack -> true
description.wurst.hack.kaboom -> true
description.wurst.hack.killauralegit -> true
description.wurst.hack.killaura -> true
description.wurst.hack.killpotion -> true
description.wurst.hack.liquids -> true
description.wurst.hack.lsd -> true
description.wurst.hack.masstpa -> true
description.wurst.hack.mileycyrus -> true
description.wurst.hack.mobesp -> true
description.wurst.hack.mobspawnesp -> true
description.wurst.hack.multiaura -> true
description.wurst.hack.nameprotect -> true
description.wurst.hack.nametags -> true
description.wurst.hack.navigator -> true
description.wurst.hack.newchunks -> true
description.wurst.hack.nobackground -> true
description.wurst.hack.noclip -> true
description.wurst.hack.nocomcrash -> true
description.wurst.hack.nofall -> true
description.wurst.hack.nofireoverlay -> true
description.wurst.hack.nohurtcam -> true
description.wurst.hack.nolevitation -> true
description.wurst.hack.nooverlay -> true
description.wurst.hack.nopumpkin -> true
description.wurst.hack.noslowdown -> true
description.wurst.hack.noweather -> true
description.wurst.hack.noweb -> true
description.wurst.hack.nuker -> true
description.wurst.hack.nukerlegit -> true
description.wurst.hack.openwateresp -> true
description.wurst.hack.overlay -> true
description.wurst.hack.panic -> true
description.wurst.hack.parkour -> true
description.wurst.hack.playeresp -> true
description.wurst.hack.portalgui -> true
description.wurst.hack.potionsaver -> true
description.wurst.hack.prophuntesp -> true
description.wurst.hack.protect -> true
description.wurst.hack.radar -> true
description.wurst.hack.rainbowui -> true
description.wurst.hack.reach -> true
description.wurst.hack.remoteview -> true
description.wurst.hack.restock -> true
description.wurst.hack.safewalk -> true
description.wurst.hack.scaffoldwalk -> true
description.wurst.hack.search -> true
description.wurst.hack.servercrasher -> true
description.wurst.hack.skinderp -> true
description.wurst.hack.sneak -> true
description.wurst.hack.snowshoe -> true
description.wurst.hack.speedhack -> true
description.wurst.hack.speednuker -> true
description.wurst.hack.spider -> true
description.wurst.hack.step -> true
description.wurst.hack.throw -> true
description.wurst.hack.tillaura -> true
description.wurst.hack.timer -> true
description.wurst.hack.tired -> true
description.wurst.hack.toomanyhax -> true
description.wurst.hack.tp-aura -> true
description.wurst.hack.trajectories -> true
description.wurst.hack.treebot -> true
description.wurst.hack.triggerbot -> true
description.wurst.hack.trollpotion -> true
description.wurst.hack.truesight -> true
description.wurst.hack.tunneller -> true
description.wurst.hack.x-ray -> true
description.wurst.other_feature.nochatreports -> true
button.wurst.nochatreports.disable_signatures -> true
button.wurst.nochatreports.re-enable_signatures -> true
description.wurst.setting.generic.attack_speed -> true
description.wurst.setting.generic.pause_attack_on_containers -> true
description.wurst.altmanager.premium -> true
description.wurst.altmanager.cracked -> true
description.wurst.altmanager.failed -> true
description.wurst.altmanager.checked -> true
description.wurst.altmanager.unchecked -> true
description.wurst.altmanager.favorite -> true
description.wurst.altmanager.window -> true
description.wurst.altmanager.window_freeze -> true
description.wurst.altmanager.fullscreen -> true
gui.wurst.altmanager.folder_error.title -> true
gui.wurst.altmanager.folder_error.message -> true
gui.wurst.altmanager.empty.title -> true
gui.wurst.altmanager.empty.message -> true
gui.wurst.nochatreports.unsafe_server.title -> true
gui.wurst.nochatreports.unsafe_server.message -> true
toast.wurst.nochatreports.unsafe_server.title -> true
toast.wurst.nochatreports.unsafe_server.message -> true
gui.wurst.nochatreports.ncr_mod_server.title -> true
gui.wurst.nochatreports.ncr_mod_server.message -> true
button.wurst.nochatreports.signatures_status -> true
description.wurst.nochatreports.message_is_reportable -> true
gui.wurst.generic.allcaps_blocked -> true
gui.wurst.generic.allcaps_allowed -> true
190

Process finished with exit code 0
I checked all Description and be all right

**Comment by JAXPLE** [comment]
`
import com.google.gson.JsonElement;
import com.google.gson.JsonObject;
import com.google.gson.JsonParser;

import java.io.FileReader;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;
import java.util.Set;

public class Main {
    public static void main(String[] args) throws Exception {
        List<String> koreaDescription = new ArrayList<>();
        List<String> englishDescription = new ArrayList<>();

        JsonParser parser = new JsonParser();

        // Load the JSON file
        FileReader fileReader = new FileReader("src/ko_KR.json");
        JsonObject jsonObject = parser.parse(fileReader).getAsJsonObject();

        // Iterate through the JSON object and extract descriptions
        Set<Map.Entry<String, JsonElement>> entries = jsonObject.entrySet();
        for (Map.Entry<String, JsonElement> entry : entries) {
            String key = entry.getKey();
            koreaDescription.add(key);
        }

        fileReader = new FileReader("src/en_us.json");
        jsonObject = parser.parse(fileReader).getAsJsonObject();

        // Iterate through the JSON object and extract descriptions
        Set<Map.Entry<String, JsonElement>> entries2 = jsonObject.entrySet();
        for (Map.Entry<String, JsonElement> entry : entries) {
            String key = entry.getKey();
            englishDescription.add(key);
        }

        int trueCnt = 0;
        for (int i = 0; i < koreaDescription.size(); i++) {
//            System.out.println(koreaDescription.get(i));
//            System.out.println(englishDescription.get(i));
            if (koreaDescription.get(i).equals(englishDescription.get(i))) {
                System.out.println(koreaDescription.get(i) + " -> " + true);
                trueCnt ++;
            }
        }
        System.out.println(trueCnt);
    }
}
`

**Comment by Alexander01998** [comment]
Hi @JAXPLE,

Thanks for the update! However, after reviewing your revised translation, it appears that most of the issues I had highlighted previously haven't actually been addressed yet. A number of strings which had severe issues - some words still in English, appear to be unchanged from the initial draft.

Could you please revisit my review comments and make the necessary fixes? Alternatively, since this pull request has been open for two months now, perhaps we should just merge the correctly translated strings and leave the ones with issues for a future PR?

Let me know how you'd like to proceed or if you have any questions.

---

Machine translation:

안녕하세요 @JAXPLE,

업데이트해 주셔서 감사합니다! 하지만 수정된 번역을 검토한 결과, 제가 이전에 강조했던 대부분의 문제가 아직 해결되지 않은 것으로 보입니다. 심각한 문제가 있었던 여러 문자열(일부 단어는 여전히 영어로 되어 있음)이 초기 초안에서 변경되지 않은 것으로 보입니다.

제 검토 의견을 다시 살펴보고 필요한 수정을 해주시겠어요? 아니면 이 풀 리퀘스트가 두 달 동안 열려 있었으므로 올바르게 번역된 문자열은 병합하고 문제가 있는 문자열은 향후 PR을 위해 남겨두는 것이 어떨까요?

어떻게 진행하고 싶은지 또는 궁금한 점이 있으면 알려주세요.

**Comment by JAXPLE** [comment]
@Alexander01998 
I left it in English because it fits better in context.
You don't have to worry about this part.
If there is a problem, I will pr again.

**Pull request closed without merging**